blog/essay/_posts/2026-08-01-just-try-the-thing.md

```
@@ -0,0 +1,212 @@
+
+> "[The optimal Claude Code Setup](https://x.com/bcherny/status/2007179832300581177)," [/grill-me](https://github.com/mattpocock/skills), and just recently, [/i-have-adhd](https://github.com/ayghri/i-have-adhd), [/adhd](https://github.com/uditakhourii/adhd), [/ponytail](https://github.com/DietrichGebert/ponytail), [/caveman](https://github.com/juliusbrussee/caveman), [ponytail *on* caveman](https://github.com/DietrichGebert/ponytail#faq), stop writing code, [write loops](https://corpwaters.substack.com/p/the-ultimate-guide-to-loops). Stop writing loops, [write graphs](https://www.aibuilderclub.com/blog/graph-engineering-vs-loop-engineering). [Delete your prompts for Opus 5](https://claude.com/blog/the-new-rules-of-context-engineering-for-claude-5-generation-models). Code review? A [cornucopia](https://coderabbit.ai) of [differently](https://greptile.com)-tuned [SaaS platforms](https://www.qodo.ai/), [skills](https://code.claude.com/docs/en/code-review), and prompts overflows every day. [Apple's got a native container engine](https://opensource.apple.com/projects/container/), maybe your agents should be in *that*. [GPT Terra, Sol, *and* Luna](https://www.mindstudio.ai/blog/what-is-gpt-5-6-sol-terra-luna-explained). Agentic memory? The [cup](https://www.superlocalmemory.com/) [runneth over](https://github.com/VictorTaelin/OptMem) and [spilleth](https://genaiunplugged.substack.com/p/give-your-ai-agents-memory-mcp-shared) onto the keyboard. [Worktrees](https://cursor.com/docs/configuration/worktrees)? How about [stacked pull requests](https://github.blog/changelog/2026-07-30-stacked-pull-requests-are-now-in-public-preview/)!
+
+It doesn't stop - if anything, it's accelerating.
+
+How do you avoid the analysis paralysis? Even having avoided it, how do you find the time to fit evaluating all these things into your day? You've got "real" work to do, too, you know. Maybe the right answer is to just keep doing what you've got now. After all, it's been working for you so far...
+
+> Just try the thing.
+
+You probably have *some* agentic engineering [harness](/garden/ai-horses.html) open somewhere.
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)